Classes/PassiveTreeView.lua

```diff
@@ -93,10 +93,12 @@ function PassiveTreeViewClass:Save(xml)
 	}
 end
 
-function PassiveTreeViewClass:Draw(build, viewPort, inputEvents)
+function PassiveTreeViewClass:Draw(build, compareSpec, viewPort, inputEvents)
 	local spec = build.spec
 	local tree = spec.tree
 
+	local isComparing = compareSpec and true or false;
```

Comment:
This can be simplified in `PassiveTreeView` by removing the `isComparing` boolean and just checking `compareSpec` everywhere instead.  They can be substituted 1-for-1

```suggestion
```